common/static/js/src/HelloWorld.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export class HelloWorld extends React.Component {
+  render() {
+    return <div>hello world {this.props.foo}</div>;
+  }
+}
```

Comment:
Will kill this file after initial signoff.

Reply:
Is there some canonical example that will remain in the code base? Where will people go to find how to use this feature correctly?

Reply:
I was planning on applying this def across the boards to all the existing React includes in platform (`StudioIndex`, `StatusAlert`, etc) once I got signoff on implementation.